backend/elasticsearch/src/main/java/org/hibernate/search/backend/elasticsearch/scope/model/impl/ElasticsearchScopeModel.java

```diff
@@ -29,15 +29,15 @@
 
 	private final Set<ElasticsearchIndexModel> indexModels;
 	private final Set<String> hibernateSearchIndexNames;
-	private final Set<URLEncodedString> elasticsearchIndexNames;
```

Comment:
They were named that way for a reason. These names are not the ones used in Hibernate Search, they were normalized to fit Elasticsearch's requirements (no uppercase, in particular).

I don't think it's wise to just call them "index names", as they could be mistaken for the names used internally in Hibernate Search.

Also, when displaying error messages, I think you should use `hibernateSearchIndexNames`, since those are the names that the user will use in the mapping and configuration files.